src/js/components/action.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import StripeCheckout from 'react-stripe-checkout';
```

Comment:
9kb unminified... best thing is, it loads checkout only when needed... no buy button? no stripe!

It won't impact load time at all.